docker/common

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+set -e
+
+PACKAGE_ROOT=$(realpath $HERE/..)
+PACKAGE_NAME=pp-sketchlib
+PACKAGE_ORG=mrcide
```

Comment:
we do have a poppunk org on dockerhub, but we could change this. We could also make a bacpop one/ What do you think is best?

Reply:
docker has changed their licencing in annoying ways, leading to rate limiting on pushes/pulls. We have applied for and got some level of exception for mrcide which is why I put it here. Very happy to use the poppunk one, but we'll need to get our robot user added to that org (there are also organisation size/user limits) and it might end up causing a few issues if we hit rate limits. But we can cross that bridge when we get to it, including getting you to organise an exception for poppunk